.github/workflows/publish.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
           echo "GIT_USER=Renku Bot" >> $GITHUB_ENV
           echo "GIT_EMAIL=renku@datascience.ch" >> $GITHUB_ENV
       - name: Build and push chart and images
-        uses: SwissDataScienceCenter/renku-actions/publish-chart@v0.5.2
+        uses: SwissDataScienceCenter/renku-actions/publish-chart@v0.5.3
```

Comment:
Further below in this file run a simple `helm lint` in a separate step before you publish the chart. It can save you a lot of trouble. If helm lint fails then you should not publish.

Reply:
p.s. helm lint should also pass on this chart rn as it is hopefully